pkg/util/util.go

```diff
@@ -2190,3 +2190,13 @@ func IsDevEnv() bool {
 	}
 	return false
 }
+
+// IsAnnotationPresent checks if an annotation is
+// is present in the Annotations map
+func IsAnnotationPresent(annotations map[string]string, name string) bool {
```

Comment:
Please use the `GetAnnotationValue` function implemented in Amit's PR [here](https://github.com/noobaa/noobaa-operator/pull/1393/files#diff-0fa449830b5dc2b274b7c95a58b5071edc23f1e44af84a99c1fa9e600767829fR1427) once it's merged